.github/workflows/tox.yml

```diff
@@ -85,3 +85,30 @@ jobs:
           if [[ "$TOXENV" == 'csslint' ]]; then npm install -g csslint; fi
       - name: Run tox
         run: python -m tox
+
+  translation:
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        run: |
+          python -m pip install -r requirements/project.txt
+      - name: Check if Portable Object Templates should have been updated
+        run: |
+          python setup.py extract_messages -t mkdocs
+          python setup.py extract_messages -t readthedocs
+          git diff --ignore-matching-lines 'POT-Creation-Date' --exit-code
+        shell: bash
+      - name: Check if Portable Object translation files should be updated on mkdocs version change
+        run: |
+          for locale in $(ls mkdocs/themes/mkdocs/locales/); do python setup.py update_catalog -t mkdocs -l ${locale}; done
+          for locale in $(ls mkdocs/themes/readthedocs/locales/); do python setup.py update_catalog -t readthedocs -l ${locale}; done
+          git diff --quiet --exit-code mkdocs/__init__.py || git diff --ignore-matching-lines 'POT-Creation-Date' --exit-code
```

Comment:
I think `git diff --quiet --exit-code mkdocs/__init__.py` will never be hit. The pull request state doesn't use uncommitted changes.

Reply:
Yep... trying to find a solution...

Reply:
It might help to see a demonstration. I suppose you could just add a commit to this PR which demonstrates that it works. Then just revert the commit. The check will still be in the the checks history.

Reply:
I'm testing on another branch here : https://github.com/ultrabug/mkdocs/actions

Reply:
https://github.com/ultrabug/mkdocs/runs/2169842544?check_suite_focus=true#step:7:21

Got it I guess

Reply:
The current implementation looks good to me as fulfilling the requirement. A bit heavy but I don't have any other approach that's strictly better.